From 3835ff38c2c4b39c3811498c3c02d76b584b6f94 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Mon, 25 Oct 2021 10:00:15 +0530
Subject: [PATCH] Revert Adding a vendor property to extract host_ip address
 vendor.intel.ipaddr Not deleting auto_hal.in as removing has issues while
 patch apply

Signed-off-by: vdanix <vishwanathx.dani@intel.com>
---
 groups/sensors/mediation/files.spec | 2 --
 groups/sensors/mediation/product.mk | 1 -
 2 files changed, 3 deletions(-)
 delete mode 100644 groups/sensors/mediation/files.spec

diff --git a/groups/sensors/mediation/files.spec b/groups/sensors/mediation/files.spec
deleted file mode 100644
index ad7141b..0000000
--- a/groups/sensors/mediation/files.spec
+++ /dev/null
@@ -1,2 +0,0 @@
-[extrafiles]
-auto_hal.in : "auto script"
diff --git a/groups/sensors/mediation/product.mk b/groups/sensors/mediation/product.mk
index f2d72e9..1bcffc6 100644
--- a/groups/sensors/mediation/product.mk
+++ b/groups/sensors/mediation/product.mk
@@ -17,4 +17,3 @@ PRODUCT_COPY_FILES += \
         frameworks/native/data/etc/android.hardware.sensor.compass.xml:vendor/etc/permissions/android.hardware.sensor.compass.xml \
         frameworks/native/data/etc/android.hardware.sensor.light.xml:vendor/etc/permissions/android.hardware.sensor.light.xml
 
-AUTO_IN += $(TARGET_DEVICE_DIR)/{{_extra_dir}}/auto_hal.in
-- 
2.33.1

